app/javascript/packs/components/navbar.tsx

```diff
@@ -52,6 +52,8 @@ export const Navbar: React.FC = () => (
         >
           Contact
         </NavLink>
+        <a href="https://twitter.com/share?text=Share Sensitive Data Securely with @hashify_app&url=Hashify.app" target="_blank">
```

Comment:
can you use the `<Navlink>` react tag and the classNames as above ?

Reply:
`<NavLink
          className="text-xl block mt-4 lg:inline-block lg:mt-0 font-title hover:text-white mr-4 navbar-link"
          to="https://twitter.com/share?text=Share Sensitive Data Securely with  @hashify_app&url=Hashify.app" 
          target="_blank"
          exact
        >
 Share on Twitter
        </NavLink>
`
        
 Something like this?? but is appending localhost too while redirecting

Reply:
ah my bad, i didn't realise `NavLink` is for internal links only. a tag is fine, just use the classNames 😄

Reply:
```
<a href="https://twitter.com/share?text=Share Sensitive Data Securely with @hashify_app&url=Hashify.app" target="_blank" className="text-xl block mt-4 lg:inline-block lg:mt-0 font-title hover:text-white mr-4 navbar-link">
```